### ISSUE TITLE
Displaying agents occupation

### ISSUE THREAD

**Issue by byroot** [buildkite/feedback#123]
In the nav bar there is the number of online agents:
<img width="124" alt="capture d ecran 2015-08-20 a 10 17 47" src="https://cloud.githubusercontent.com/assets/44640/9385507/d976ce20-4724-11e5-9456-8ae24864c63b.png">

It would be very useful to also display the number of busy agents, like `342/582`.

/nitpick

**Comment by keithpitt** [comment]
Yup! We've got some designs we're working on that includes that number :)
